jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -288,6 +288,10 @@ def _uncertainty(result):
             # don't need these if statements
             if function == "Line Flux":
                 flux_unit = spec_subtracted.flux.unit
+                add_flux = False
+                if str(flux_unit) == '':
+                    add_flux = True
+                    flux_unit = u.Unit(self.spectrum_viewer.state.y_display_unit)
```

Comment:
```suggestion
                if str(flux_unit) == '':
                    add_flux = True
                    flux_unit = u.Unit(self.spectrum_viewer.state.y_display_unit)
                else:
                    add_flux = False
```

Reply:
This is added in the most recent commit!